src/Service/RestRequestHelper.php

```diff
@@ -90,7 +108,8 @@ public function getOptionsFromRequest(Request $request): RestRequestOptions
 
     /**
      * Returns whether this is REST configured request.
-     * Only works after setting Options – use resolveRestRequestOptions before that
+     * Only works after setting Options – use either
```

Comment:
We should not recommend to use internal methods, which is our bundle's implementation detail. The outside code (developer that we're communicating this message to) should not call that at all.

What we could communicate here, is that this will work only after request listener for annotated requests, and only after controller listener for other requests. Or something similar.